### ISSUE TITLE
Segmentation fault when running "python train.py"

### ISSUE THREAD

**Issue by txf201604** [xingyul/flownet3d#53]
root@f9adda5586c3:/mnt/txf/codes/FlowNet3D# python train.py
pid: 22198
WARNING:tensorflow:From /mnt/txf/codes/FlowNet3D/model_concat_upsa.py:16: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From train.py:83: The name tf.train.exponential_decay is deprecated. Please use tf.compat.v1.train.exponential_decay instead.

WARNING:tensorflow:From train.py:102: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

--- Get model and loss
WARNING:tensorflow:From /mnt/txf/codes/FlowNet3D/model_concat_upsa.py:39: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

Segmentation fault (core dumped)

My running environment configuration as follow:
GPU: GTX 1080
CUDA: CUDA10.0
Tensorflow: Tensorflow-gpu-1.14.0

I get a segmentation fault when running the train.py gets to the function https://github.com/xingyul/flownet3d/blob/master/tf_ops/sampling/tf_sampling.py#L56. I am not sure what could be causing the issue as I got to compile with no problems the code previously.

**Comment by txf201604** [comment]
OK. I get it!
Thank you!
The issue is with the version of the g++ (7.4.0) on Ubuntu 18.04.
Simply installing the g++-4.8 and using it to build the tf_ops solved the issue.
